src/wallet/wallet.cpp

```diff
@@ -943,6 +944,15 @@ bool CWallet::LoadUnifiedCaches()
         }
     }
 
+    // Restore decrypted Orchard notes.
+    for (const auto& [txid, walletTx] : mapWallet) {
```

Comment:
`txid` is not used. See if there is a way to iterate only over the values of `mapWallet`. Not-too-blocking as this syntax is nicer than the iterator syntax.